File: maui/PhotoApp/Models/Photo.cs
using System.Text.Json.Serialization;

namespace PhotoApp;

public class Photo
{
    public int albumId { get; set; }
    public int id { get; set; }
    public string title { get; set; }
    public string url { get; set; }
    public string thumbnailUrl { get; set; }
}

[JsonSerializable(typeof(List<Photo>))]
internal sealed partial class PhotoContext : JsonSerializerContext
{
}

File: android/app/src/main/java/com/whackylabs/photoapp/Photo.kt
package com.whackylabs.photoapp

import java.io.Serializable

//import kotlinx.serialization.Serializable

data class Photo(
    val albumId: Int?,
    val id: Int?,
    val title: String?,
    val url: String?,
    val thumbnailUrl: String?
) : Serializable
